File: PC.Quiz.Domain/Entities/Question.cs
namespace PC.Quiz.Domain.Entities
{
    using PC.Quiz.Domain.Enums;
    using System.Collections.ObjectModel;

    public class Question
    {
        public long Id { get; set; }

        public string Title { get; set; } = string.Empty;

        public int Points { get; set; }

        public QuestionType Type { get; set; }

        public ICollection<EntryAnswer> EntryAnswers { get; set; } = new Collection<EntryAnswer>();

        public ICollection<AnswerOption> AnswerOptions { get; set; } = new Collection<AnswerOption>();
    }
}

File: pc.quiz.client/src/shared/types/entities/Question.ts
import { QuestionType } from "../enums/QuestionType";
import { AnswerOption } from "./AnswerOption";
import { EntryAnswer } from "./EntryAnswer";

export interface Question {
    id: number,
    title: string,
    type: QuestionType,
    answerOptions: AnswerOption[]
    entryAnswers: EntryAnswer[]
}